src/DFTK.jl

```diff
@@ -88,6 +88,7 @@ export spin_density
 export ρ_from_total_and_spin
 include("densities.jl")
 include("interpolation_transfer.jl")
+export transfer_matrix
```

Comment:
This function is not like the others... Build transfer matrix or something like that?

Reply:
Good point. Though our prefix is `compute_` ...